airflow/jobs/triggerer_job.py

```diff
@@ -484,9 +484,14 @@ async def create_triggers(self):
         while self.to_create:
             trigger_id, trigger_instance = self.to_create.popleft()
             if trigger_id not in self.triggers:
+                task_instance: TaskInstance = trigger_instance.task_instance
+                dag_id = task_instance.dag_id
+                run_id = task_instance.run_id
+                task_id = task_instance.task_id
+                map_index = task_instance.map_index
                 self.triggers[trigger_id] = {
                     "task": asyncio.create_task(self.run_trigger(trigger_id, trigger_instance)),
-                    "name": f"{trigger_instance!r} (ID {trigger_id})",
+                    "name": f"{dag_id}/{run_id}/{task_id}/{map_index} (ID {trigger_id})",
```

Comment:
Do we want to include `try_number` here? It is a part of `key` on `TaskInstance`. (Or maybe we should use `key` directly?

Reply:
I think including the `try_number` is a good idea